fasttrack/cli.py

```diff
@@ -247,6 +339,7 @@ def _load_data_and_meta(dummies: dict[str, str]) -> Tuple[pd.DataFrame, YAMLMeta
         if not success:
             continue
 
+        # NOTE: harmonization is not done in ETL, but here in fast-track for technical reasons
```

Comment:
Yes, this is not ideal.

Reply:
It's not yet clear what will authors prefer and how should we handle preprocessing from raw data to data saved in the snapshot. Only time will tell...